<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registro_cliente</class>
 <widget class="QDialog" name="Registro_cliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>490</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #9fd5d1;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Bienvenido">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Segoe Print&quot;;
background: transparent;</string>
    </property>
    <property name="text">
     <string>¡Registrate!</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>301</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="Nombre">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
background: transparent;
border bottom: rgb(255, 255, 255);

</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Nombre(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ApellidoPat">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(0,0,0);
background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Apellido paterno</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ApellidoMat">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(0,0,0);
background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Apellido materno</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Correo">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(0,0,0);
background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Correo electronico</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Contrasea">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(0,0,0);
background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Contraseña (min. 6 caracteres)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Contrasea_2">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(0,0,0);
background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string>Confirmar contraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Telefono">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(0,0,0);
background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Teléfono</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="NumeroTarjeta">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(0,0,0);
background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>16</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Número de tarjeta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Titular">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(0,0,0);
background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Titular de la tarjeta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Matricula">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Times New Roman&quot;;
color: rgb(0,0,0);
background: transparent;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Matricula (opcional)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Imagenes.qrc">:/Imagenes/Imagenes/Icon_2_0.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="CerrarSesion">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>440</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#DB7B65;
font: 14pt 75 &quot;Times New Roman&quot;;
hover!pressed:border: 5px solid green;

</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Guardar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt 75 &quot;Times New Roman&quot;;
hover!pressed:border: 5px solid green;
background-color: #9BC499;</string>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
